init_files/v1/iplant_backup.sh

```diff
@@ -1,346 +0,0 @@
-#!/bin/bash
-
-################################################################
-#
-#  Script to backup or retrieve data between an Atmosphere
-#  instance and iPlant Datastore
-#  
-#  iplant_backup.sh -- Version 1.0
```

Comment:
Is this iplant_backup script stored elsewhere? I believe that this was being used by end-users at one point in time...